packages/core/src/Serialization/Common/CborSet.ts

```diff
@@ -0,0 +1,94 @@
+import { CborReader, CborReaderState, CborTag, CborWriter } from '../CBOR';
+import { HexBlob } from '@cardano-sdk/util';
+
+export class CborHexType<T extends string> {
```

Comment:
We represent these types as Hex on core types because we want to have them in string format, but in the CDDL these are Bytestrings. I would move this to its own file and rename it. Looking at the CDDL and the way you are using it, it seems we can call this `Hash` i, e:

`CborSet<Hash<Crypto.Ed25519KeyHashHex>>)`